app/Entities/Place/PlaceCategory.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Hedonist\Entities\Place;
+
+use Illuminate\Database\Eloquent\Model;
+use Prettus\Repository\Contracts\Transformable;
+use Prettus\Repository\Traits\TransformableTrait;
+
+class PlaceCategory extends Model implements Transformable
```

Comment:
Please remove `implements Transformable` and TransformableTrait. We will use special Presenter classes for transforming models.